src/models/media.ts

```diff
@@ -0,0 +1,222 @@
+/*
```

Comment:
Not convinced this set of *interfaces* is the same kind of thing as the other stuff in `models`, which are (supposedly) higher-level classes for managing objects such as rooms and events. Could we find another home for it?